layout

```diff
@@ -0,0 +1,244 @@
+####### EXISTING OUTLINE
+
+X=delete  G=gallery (or example)  ?=not sure where this would go
+
+    Tutorials
+?       tut: Quick Start Guide
+G       tut: Trial API: PyTorch MNIST Tutorial
+G       tut: Trial API: TensorFlow Keras Fashion MNIST Tutorial
+        tut: Accessing Data
+X       tut: Native API Tutorial
+        tut: Notebooks Tutorial
+        tut: Organizing Models in the Model Registry
+
+    How-To Guides
+        how: Install Determined CLI
+        how: Hyperparameter Tuning
+        how: Distributed Training
+        how: Environment Configuration
+        how: Data Layer
+        how: How to Install Determined
+        how: Notebooks
+        how: TensorBoard
+        how: Using Checkpoints
+X       how: IDE setup
+        how: Using the Determined REST APIs
+
+    Topic Guides
+        top: Benefits of Determined
+        top: Determined on Kubernetes
+        top: Specifying Custom Pod Specs
+        top: Effective Distributed Training
+        top: Experiment Lifecycle
+        top: Exporting Models and Checkpoints
+        top: Hyperparameter Tuning Defined
+        top: Hyperparameter Tuning With Determined
+        top: Model Definitions
+X       top: Native API
+X       top: Trial API
+        top: Best Practices for Model Definitions
+        top: Optimizing Distributed Training
+        top: Reproducibility
+        top: Scheduling
+        top: System Architecture
+        top: Determined on AWS
+        top: Determined on GCP
+        top: Elastic Infrastructure
+        top: Dynamic Agents on AWS
+        top: Dynamic Agents on GCP
+        top: Terminology and Concepts
+X       top: User Interfaces
+        top: Users
+X       top: YAML Topic Guide
+
+    Reference
+        ref: API Reference
+        ref: Cluster Configuration
+        ref: Command-line Interface
+        ref: Tensorboard and Notebook Configuration
+        ref: Configuration Templates
+        ref: Experiment Configuration
+        ref: Helm Chart Configuration
+        ref: Open Source Licenses
+
+    Frequently Asked Questions
+    Examples
+    Release Notes
+
+
+####### PROPOSED REORG
+
+# Guiding principles of reorg:
+#  - topical organization
+#  - fewer pages = better ctrl-f and less hierarchy to navigate
+
+N=new content   *=doc-level item   ?=not sure if we have this already
+
+    Quick Start
+N     * Running Your First Experiment
+            prereqs:
+                docker works
+            goals:
+                have working installation (docker method)
+                have cli installed
+                    how: Install Determined CLI
+                hands-on use of cli and webui (tensorboard?), and nothing else
+            details:
+                use one-variable model (copy/paste rather than download code)
+N     * Porting A PyTorch Model
```

Comment:
I think we should avoid the phrase "porting". Alternatives:

"Building Your First PyTorch Model in Determined"
"Using PyTorch with Determined"

etc.

Reply:
> "Building Your First PyTorch Model in Determined"

To me this sounds too much like "starting from scratch", I'd be afraid people who needed to port a model might not recognize the title as what they need

What about "Starting with an existing PyTorch Model"?

Or "Train an existing PyTorch model with Determined"?

Or "Bring an existing PyTorch model into Determined"?

Or "Structuring a PyTorch model for Determined"?

Reply:
"Train an existing PyTorch model with Determined" is my favorite